### ISSUE TITLE
vertx-jooq-rx-jdbc:  java.lang.NoSuchMethodError

### ISSUE THREAD

**Issue by paninics** [jklingsporn/vertx-jooq#97]
I've this simple piece of code:
```
DefaultConfiguration config = new DefaultConfiguration();
config.setConnection(getConnection());
config.set(SQLDialect.POSTGRES);
Vertx vertx = Vertx.vertx(); // io.vertx.reactivex.core.Vertx
JDBCRXGenericQueryExecutor queryExecutor = new JDBCRXGenericQueryExecutor(config, vertx);
SomeDao dao = new SomeDao(config, vertx);
dao.findOneById((short) 1);
```
And I get this error:

```
Exception in thread "main" java.lang.NoSuchMethodError: io.vertx.reactivex.core.Vertx.rxExecuteBlocking(Lio/vertx/core/Handler;)Lio/reactivex/Single;
	at io.github.jklingsporn.vertx.jooq.rx.jdbc.JDBCRXGenericQueryExecutor.executeBlocking(JDBCRXGenericQueryExecutor.java:35)
	at io.github.jklingsporn.vertx.jooq.rx.jdbc.JDBCRXQueryExecutor.findOne(JDBCRXQueryExecutor.java:31)
	at io.github.jklingsporn.vertx.jooq.rx.jdbc.JDBCRXQueryExecutor.findOne(JDBCRXQueryExecutor.java:15)
	at io.github.jklingsporn.vertx.jooq.shared.internal.AbstractVertxDAO.findOneByCondition(AbstractVertxDAO.java:98)
	at io.github.jklingsporn.vertx.jooq.shared.internal.AbstractVertxDAO.findOneById(AbstractVertxDAO.java:93)
```

My initial though was some stale vertx binaries are being loaded. Hence I cleaned up my maven cache and still see the same error. Upon further debugging, it appears some incompatible types are surfacing - see attached screenshot. Any thoughts? My environment: `jooq: 3.11.9, vertx jooq: 4.0.0, vertx: 3.6.2`

![incompatible type](https://user-images.githubusercontent.com/23061690/51933776-ebe23000-23b6-11e9-9082-a9c8809f4d65.png)
.

**Comment by jklingsporn** [comment]
This will be fixed in 4.1. At some point during vertx 3.6.x releases the
return type of the rxBlocking method has been changed from Single to Maybe.

Am Di., 29. Jan. 2019, 20:23 hat paninics <notifications@github.com>
geschrieben:

**Comment by jklingsporn** [comment]
Fixed.
